src/model/AssetsCompiler.ts

```diff
@@ -107,10 +109,25 @@ export class AssetsCompiler {
         private readonly runConfig: IAssetsCompilerConfiguration,
         private readonly repositoryDir: string
     ) {
+        if (AssetsCompiler.shouldUseAscLocal()) {
+            console.warn(
+                cwarn`Starting from the cplace release 23.2 you should be using @cplace/asc-local instead of @cplace/asc!`
+            );
+        }
         this.repositoryName = path.basename(repositoryDir);
         this.projects = this.setupProjects();
     }
 
+    private static shouldUseAscLocal(): boolean {
+        const version = CplaceVersion.get();
+        const isMajorHigherThen23 = version.major > 23;
+        const isMinorHigherOrEqual2 =
+            version.major === 23 && version.minor >= 2;
```

Comment:
Could we make this like
```ts
const isAtLeast23_2 = (version.major === 23 && version.minor >= 2) || version.major > 23;
```
?
Doesn't have to be, but we should at least rename `isMinorHigherOrEqual2` as it also checks the major version. 🤔